monkey/infection_monkey/network/tools.py

```diff
@@ -189,97 +158,6 @@ def tcp_port_to_service(port):
     return "tcp-" + str(port)
 
 
-def traceroute(target_ip, ttl=64):
```

Comment:
@VakarisZ Are we using traceroute anywhere anymore? If not, can we get rid of everything that references the binaries (e.g. deployment scripts and release documentation).